apps/emqx_connector/src/emqx_connector_http.erl

```diff
@@ -384,14 +384,15 @@ on_query_async(
 
 on_get_status(_InstId, #{pool_name := PoolName, connect_timeout := Timeout} = State) ->
     case do_get_status(PoolName, Timeout) of
-        true ->
-            connected;
-        false ->
+        ok ->
+            {connected, State};
+        {error, Reason} ->
             ?SLOG(error, #{
                 msg => "http_connector_get_status_failed",
```

Comment:
Tiny nit: this log seems repeated with l. 406?  Maybe we could keep only the latter and rename that message to `http_connector_get_status_failed`, since we have more context at that line?